ui/app/components/mount-backend-form.js

```diff
@@ -95,7 +95,8 @@ export default class MountBackendForm extends Component {
 
   @task
   @waitFor
-  *mountBackend() {
+  *mountBackend(event) {
+    event.preventDefault();
```

Comment:
There were quite a few instances where a form was submitted and the form action was firing which was causing a content security policy error.

Reply:
I've seen a lot of those console errors, good to know that's where that is coming from.